stylesheets/components/slider.css

```diff
@@ -0,0 +1,16 @@
+@import url(/stylesheets/main.css);
```

Comment:
You might not have to import this here as:
All of your component stylesheets are imported onto the main.css file and that is the same file you are using in your HTML so this can be removed.

Reply:
okay ty done